### PR TITLE
Production_Bugfix: consolidate promotion message

### PR DESCRIPTION
**Pull request by wannacfuture**
Resolves https://github.com/ubiquibot/production/issues/23
QA example: https://github.com/wannacfuture/Battleship/issues/23#issuecomment-1637239214

### CODE REVIEW

**Review by 0xcodercrane — APPROVED** [review]

File: src/handlers/comment/handlers/index.ts [hunk 1]
```diff
@@ -45,11 +45,8 @@ export const issueClosedCallback = async (): Promise<void> => {
   if (!issue) return;
   try {
     const comment = await handleIssueClosed();
-    if (comment) await addCommentToIssue(comment, issue.number);
-    await addCommentToIssue(
-      `If you enjoy the DevPool experience, please follow <a href="https://github.com/ubiquity">Ubiquity on GitHub</a> and star <a href="https://github.com/ubiquity/devpool-directory">this repo</a> to show your support. It helps a lot!`,
-      issue.number
-    );
+    const promotionCommnet = `\n<h6>If you enjoy the DevPool experience, please follow <a href="https://github.com/ubiquity">Ubiquity on GitHub</a> and star <a href="https://github.com/ubiquity/devpool-directory">this repo</a> to show your support. It helps a lot!</h6>`;
```
Comment (pavlovcik) [inline src/handlers/comment/handlers/index.ts:48]: This variable is spelled wrong. Also, I'm not sure if this was originally specified, but I think this should be configurable by every repo config.

Comment (wannacfuture) [reply src/handlers/comment/handlers/index.ts:48]: Which one should be configurable? the site links?

Comment (wannacfuture) [reply src/handlers/comment/handlers/index.ts:48]: Opened new pr.

Comment (pavlovcik) [reply src/handlers/comment/handlers/index.ts:48]: Thank you. Next time you should link to this issue, not the other one!

**Pull request merged**